contracts/bridge/Bridge.sol

```diff
@@ -23,11 +23,12 @@ contract Bridge is IERC20BridgeReceiver, IERC721BridgeReceiver, BridgeFee, Bridg
     address public counterpartBridge;
     bool public isRunning;
 
+
```

Comment:
```suggestion
```

Reply:
This PR will be rebased after #126 and the PR fixes it.